### ISSUE TITLE
6.3 branch doesn't seem to be functioning correctly

### ISSUE THREAD

**Issue by solomon23** [robb/Cartography#104]
I pulled down the 6.3 last week and noticed there seemed to be a retain cycle in it, so views that used it weren't getting deinit'ed.  I updated to the latest today and it seems really broken.

I'm using this code to add a view:
```
        addSubview(_title)
        _title.font = UIFont.boldSystemFontOfSize(22)
        _title.textAlignment = .Center
        layout(_title) {
            $0.left == $0.superview!.left
            $0.right == $0.superview!.right
            $0.height == $0.superview!.height / 4
            $0.top == $0.superview!.top + 20
        }
```
And instead of resizing the content of the label, it seems to be resizing the parent view

**Comment by solomon23** [comment]
Looks like the latest is still leaking. Here's a sample project it has a branch for the old and new:
https://github.com/solomon23/CartographyLeak
https://github.com/solomon23/CartographyLeak/tree/latest

If you run it you can see if you click the red view a deinit never happens

I think the other issue I'm facing is that the newer cartography sets 
```
setTranslatesAutoresizingMaskIntoConstraints(false)
```
On both views where the old version just set it on the left view.  The problem is I have something I want to autolayout set inside a view constructed with frames ( because I want to animate it without autolayout ) so that breaks things for me

**Comment by solomon23** [comment]
Looks like this was fixed as #99 which hasn't made it over to the 6.3 branch it seems

**Comment by robb** [comment]
Thanks, I updated the `xcode6-3` branch.
